source/portable/NetworkInterface/xilinx_ultrascale/x_emacpsif_dma.c

```diff
@@ -662,7 +665,10 @@ XStatus init_dma( xemacpsif_s * xemacpsif )
         #if ( ipconfigDRIVER_INCLUDED_TX_IP_CHECKSUM != 0 )
             value |= XEMACPS_DMACR_TCPCKSUM_MASK;
         #else
-        #warning Are you sure the EMAC should not calculate outgoing checksums?
+            #if ( ipconfigPORT_SUPPRESS_WARNING != 0 )
+            #warning Are you sure the EMAC should not calculate outgoing checksums?
```

Comment:
Indentation

Reply:
Updated. It's modified by uncrustify. I added '{}' to make indentation right.